### ISSUE TITLE
Numeric datatype isn't working in 10.5

### ISSUE THREAD

**Issue by FransdeJong** [umbraco/Umbraco-CMS#14117]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

10.5.0

### Bug summary

A numeric property editor won't save decimals since 10.5.0.

### Steps to reproduce

1. Install 10.5.0
2. Add a doctype and allow at root
3. Add a property, select numeric and set step to 0.01
4. Add content and try and save 1 in the property (this works)
5. Now save the same property with 1.01 (this fails)

**Comment by ronaldbarendse** [comment]
The default Numeric data type uses `Umbraco.Integer` as property editor, so only whole numbers are allowed. But even with that in mind, there's a few issues I can quickly spot:
- The configuration values (min, step size and max) should only allow whole numbers for the `Umbraco.Integer` property editor;
- There's no validation error shown when the entered value isn't correct;
- Changing the property editor on the existing Numeric data type doesn't seem to clear the caches and requires a restart to start accepting the decimal values 🤯

**Comment by FransdeJong** [comment]
Arnold just pointed me at the integer part.
I totally overlooked it since I remember only one numeric being present in V8?
Also the lack of feedback didn't point me in the right direction.

**Comment by elit0451** [comment]
Yes, providing some sort of validation message would be very useful in this case. Everything that @ronaldbarendse pointed out is a good candidate for an up-for-grabs fix.

**Comment by nielslyngsoe** [comment]
As mentioned, the affected versions of this issue are no longer in support, I will close this issue.

Here is a screenshot of how it is handled in v.17, as a reference:

<img width="881" height="258" alt="Image" src="https://github.com/user-attachments/assets/94f9762c-e1fa-4021-8ef0-25152219a8c4" />

Thanks for the report, and pleace keep such coming, they are very important for authors so surely something we should handle, so if similar comes up please tag me and I will dedicate attention to it.
